src/pipx/main.py

```diff
@@ -27,32 +28,49 @@ def print_version() -> None:
     print(__version__)
 
 
+def indented_wrap(text, subsequent_indent="", split="\n", **kwargs):
+    text = textwrap.dedent(text).strip()
+    width = shutil.get_terminal_size().columns - 2
+    return "\n".join(
+        [
+            textwrap.fill(line, width=width, subsequent_indent=subsequent_indent)
+            for line in text.splitlines()
+        ]
+    )
+
+
 SPEC_HELP = textwrap.dedent(
     """The package name or specific installation source passed to pip.
     Runs `pip install -U SPEC`.
     For example `--spec mypackage==2.0.0` or `--spec  git+https://github.com/user/repo.git@branch`
     """
 )
+
 PIPX_DESCRIPTION = textwrap.dedent(
     f"""
-Install and execute apps from Python packages.
+        Install and execute apps from Python packages.
 
-Binaries can either be installed globally into isolated Virtual Environments
-or run directly in an temporary Virtual Environment.
+        Binaries can either be installed globally into isolated Virtual Environments
+        or run directly in an temporary Virtual Environment.
 
-Virtual Environment location is {str(constants.PIPX_LOCAL_VENVS)}.
-Symlinks to apps are placed in {str(constants.LOCAL_BIN_DIR)}.
+        Virtual Environment location is {str(constants.PIPX_LOCAL_VENVS)}.
+        Symlinks to apps are placed in {str(constants.LOCAL_BIN_DIR)}.
 
-Optional Environment Variables:
-PIPX_HOME: Overrides default pipx location. Virtual Environments
-will be installed to $PIPX_HOME/venvs.
-PIPX_BIN_DIR: Overrides location of app installations. Apps are symlinked
-or copied here.
-USE_EMOJI: Override emoji behavior. Default value varies based on platform.
-PIPX_DEFAULT_PYTHON: Overrides default python used for commands.
-"""
+        """
+)
+PIPX_DESCRIPTION += "\n"
+PIPX_DESCRIPTION += indented_wrap(
+    """
+        optional environment variables:
+          PIPX_HOME:           Overrides default pipx location. Virtual Environments will be installed to $PIPX_HOME/venvs.
+          PIPX_BIN_DIR:        Overrides location of app installations. Apps are symlinked or copied here.
+          USE_EMOJI:           Override emoji behavior. Default value varies based on platform.
+          PIPX_DEFAULT_PYTHON: Overrides default python used for commands.
+    """,
+    " " * 23,  # magic number 23 matches the indent of the env descriptions.
```

Comment:
How about making this 24, so it exactly matches the indent of the `optional arguments` descriptions?

Reply:
Done.  

Also changes USE_EMOJI to use "Overrides" to match the other 3.
Though I might argue that they would all read better as "Override"

Reply:
- [x] How about making this 24, so it exactly matches the indent of the optional arguments descriptions?
- [x] ( and adding a space to before the start of the optional environment variables description start.)
- [x] And while you're add it, what about removing the colons after the environment variables?

Current alignment of indents in wrapped and wide formats:

```
Binaries can either be installed globally into isolated Virtual Environments
or run directly in an temporary Virtual Environment.

Virtual Environment location is /Users/agrangaard/.local/pipx/venvs.
Symlinks to apps are placed in /Users/agrangaard/.local/bin.

optional environment variables:
  PIPX_HOME             Overrides default pipx location. Virtual Environments
                        will be installed to $PIPX_HOME/venvs.
  PIPX_BIN_DIR          Overrides location of app installations. Apps are
                        symlinked or copied here.
  USE_EMOJI             Overrides emoji behavior. Default value varies based
                        on platform.
  PIPX_DEFAULT_PYTHON   Overrides default python used for commands.

optional arguments:
  -h, --help            show this help message and exit
  --version             Print version and exit
```

```
Virtual Environment location is /Users/agrangaard/.local/pipx/venvs.
Symlinks to apps are placed in /Users/agrangaard/.local/bin.

optional environment variables:
  PIPX_HOME             Overrides default pipx location. Virtual Environments will be installed to
                        $PIPX_HOME/venvs.
  PIPX_BIN_DIR          Overrides location of app installations. Apps are symlinked or copied here.
  USE_EMOJI             Overrides emoji behavior. Default value varies based on platform.
  PIPX_DEFAULT_PYTHON   Overrides default python used for commands.

optional arguments:
  -h, --help            show this help message and exit
  --version             Print version and exit
```

Look about right?